ilc/server/registry/Registry.js

```diff
@@ -93,4 +98,23 @@ module.exports = class Registry {
 
         return res.data;
     };
+
+    /**
+     * Returns new object with routes list for current domain
+     * and without "supportedDomains" data and property "domainId" inside every "route"
+     */
+    #filterConfigByDomain = (config, domain) => {
+        const { supportedDomains, ...configWithoutSupportedDomains } = config;
+
+        const currentDomainId = supportedDomains.find(n => n.value === domain)?.id;
+        configWithoutSupportedDomains.routes = configWithoutSupportedDomains.routes.reduce((acc, route) => {
+            if (currentDomainId === route.domainId) {
```

Comment:
From what I understand here - if `currentDomainId !== undefined` only routes w/o domain assignment will be returned here.